ml-agents/mlagents/trainers/policy/tf_policy.py

```diff
@@ -353,6 +353,16 @@ def retrieve_previous_action(self, agent_ids: List[str]) -> np.ndarray:
                 action_matrix[index, :] = self.previous_action_dict[agent_id]
         return action_matrix
 
+    def retrieve_previous_action_single(self, agent_id: str) -> np.ndarray:
```

Comment:
I think this method needs to be moved to `policy.py` with the other `retrieve` methods (#4254).